pytest_plugins/infra_dependent_markers.py

```diff
@@ -0,0 +1,20 @@
+# Custom infra dependent markers for selection and deselection of tests
+
+
+def pytest_configure(config):
+    """Register custom markers to avoid warnings."""
+    markers = []
+    markers.extend(infra_markers())
```

Comment:
Why all of this function calling to return a list?  You can just define a variable with the list at the module level, and iterate over it here without creating a new variable scope.